OpenRA.Game/FieldSaver.cs

```diff
@@ -121,7 +122,7 @@ public static string FormatValue(object v)
 					var formattedKey = FormatValue(key);
 					var formattedValue = FormatValue(value);
 
-					result += "{0}: {1}{2}".F(formattedKey, formattedValue, Environment.NewLine);
+					result += "{0}{1}: {2}{3}".F("\t\t", formattedKey, formattedValue, Environment.NewLine);
```

Comment:
This will break code that relies on the existing behaviour (the comment above mentions documentation generation).